classes/class-wc-rest-connect-address-normalization-controller.php

```diff
@@ -56,20 +56,29 @@ public function register_routes() {
 				'permission_callback' => array( $this, 'update_items_permissions_check' ),
 			),
 		) );
+
+		register_rest_route( $this->namespace, '/' . $this->rest_base . '/accept', array(
```

Comment:
This is going to conflict hard with https://github.com/Automattic/woocommerce-services/pull/925 . If my PR is merged first I'll give you a hand on fixing the conflicts :)